docs/guides/run-jobs-session.mdx

```diff
@@ -114,7 +114,7 @@ with Session(service=service, backend=backend, max_time="25m"):
   ...
 ```
 
-There is also an interactive timeout value (ITTL, or interactive time to live) that cannot be configured.  If no session jobs are queued within that window, the session is temporarily deactivated. To determine a session's ITTL, follow the instructions in [Determine session details](#session-details) and look for the `interactive_timeout` value.
+There is also an interactive time to live (iTTL) value that cannot be configured.  If no session jobs are queued within that window, the session is temporarily deactivated. To determine a session's ITTL, follow the instructions in [Determine session details](#session-details) and look for the `interactive_timeout` value.
```

Comment:
Why lowercase? If you make this change, we need to check all use cases of ITTL, like the following sentence in this paragraph

Reply:
They just need to be consistent, I need to go back and check them all.